### DISCUSSION TITLE
Dependabot outage?

### DISCUSSION THREAD

**Discussion by sgerrand** [community/community#35186] [Code Security] [answered]
I haven't seen Dependabot updates for a while on our organisation repositories, with public and private, and @dependabot isn't responding to commands any more. Is this functionality affected by an outage?

**Comment by jeffwidman** [comment, marked as answer]
No known issues. Start by checking for errors in the job logs: `https://github.com/<org>/<repo>/network/updates`

If that doesn't provide any clues, then file a private support ticket including your PR's where dependabot isn't responding to commands and we will try to debug it.

**Reply by sgerrand** [reply]
Thanks, that's useful and apologies for the noise.
